BlitzBoard.xcodeproj/project.pbxproj

```diff
@@ -641,6 +641,7 @@
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_TEAM = K59C97HYKW;
```

Comment:
why do you need to change the development team?

Reply:
I need it to run my app on my personal device